hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metrics/HoodieMetrics.java

```diff
@@ -360,23 +360,23 @@ public void updateArchiveMetrics(long durationInMs, int numInstantsArchived) {
 
   public void updateFinalizeWriteMetrics(long durationInMs, long numFilesFinalized) {
     if (config.isMetricsOn()) {
-      LOG.info(String.format("Sending finalize write metrics (%s=%d, %s=%d)", DURATION_STR, durationInMs,
-          FINALIZED_FILES_NUM_STR, numFilesFinalized));
+      LOG.debug("Sending finalize write metrics ({}={}, {}={})", DURATION_STR, durationInMs,
```

Comment:
Flagging this class: I changed the level to debug since we have metrics here the logging at info level seemed redundant.